_positions/software-developer.md

```diff
@@ -31,3 +31,5 @@ Using modern, open source web frameworks and tools, our developers (front end, b
 - Participate in or lead (as necessary) an open dialogue with representatives from our partner agencies in implementing modern development standards, including transparency, user-centered design, and agile methodologies; understand and communicate the "why" of these standards, not just the "what."
 - Demonstrate resilience in the face of bureaucracy: When faced with a hurdle, determine the root cause and address that in addition to the hurdle itself.
 - Push the boundaries of what is possible within government while maintaining an understanding of the complexities of the organization; push back against the things that can be pushed back against and take a longer view for the things that can't.
+
+##[Apply to join 18F]({{ site.baseurl }}/pages/apply.html)
```

Comment:
Same questions as in #207/209. (No need to answer in each PR, just in #207 or #209.)